### ISSUE TITLE
NixOS 19.09.891 breaks manually configured wired connection in NetworkManager

### ISSUE THREAD

**Issue by alexeymuranov** [NixOS/nixpkgs#71655]
**Describe the bug**

After my last `nixos-rebuild` on NixOS 19.09 (to `nixos-19.09.891.80b42e630b2`), NetwokManager is malfunctioning in a strange way, and in particular selecting a manually configured connection does not allow internet access in Firefox or Chromium.

Here are some symptoms:

1. When selecting a manually configured wired connection in NetworkManager (on Gnome 3 desktop), there is no Internet access in browsers. However, Openweather Gnome Shell extension succeeds in updating the current weather status. Running `ping` gives inconsistent results:
    ```console
    $ ping example.com
    PING example.com (93.184.216.34) 56(84) bytes of data.
    64 bytes from 93.184.216.34: icmp_seq=1 ttl=47 time=82.4 ms
    From 169.254.19.230 icmp_seq=2 Destination Host Unreachable
    From 169.254.19.230 icmp_seq=3 Destination Host Unreachable
    ^C
    --- example.com ping statistics ---
    254 packets transmitted, 1 received, +2 errors, 99.6063% packet loss, time 282197ms
    rtt min/avg/max/mdev = 82.380/82.380/82.380/0.000 ms, pipe 4
    ```
    on a different attempt:
    ```console
    $ ping example.com
    ping: example.com: System error
    ```
2. When trying to turn off wired connection in the NetworkManager applet (Gnome 3), the wired connection is shown as Off for a short time, and then reconnects by itself to some `enp0s31f6` which was not there before. Internet still does not work.

**To Reproduce**

Booting to `nixos-19.09.891.80b42e630b2` and selecting a manually configured connection in NetworkManager (manual IPv4, manual DNS) reproduces the problem each time for me.

For the second symptom (impossibility to turn wired connection off), just try to turn wired connection off.

**Expected behavior**

1. Internet connection should work.
2. When wired connection is turned off, it should stay off.

**Screenshots**

![Screenshot from 2019-10-22 11-48-42](https://user-images.githubusercontent.com/712698/67279570-8f3a1e80-f4cb-11e9-8e3c-94e3e30b1b61.png)

**Additional context**

1. Automatic wired connection and WiFi work.
2. Booting previous generation `19.09.840.8bf142e001b` solves the problem.

**Metadata**

```console
$ nix run nixpkgs.nix-info -c nix-info -m
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.80-hardened, NixOS, 19.09.891.80b42e630b2 (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3`
 - channels(alexey): `"nixos-17.03-17.03.1949.78e9665b48f, nixos-17.09-17.09.3238.e984f9e48e1"`
 - channels(root): `"nixos-19.09.891.80b42e630b2"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by loogol** [comment]
Same in KDE. Connection works about 1 minute, then @Destination Host Unreachable@. after reenabling connection in plasma-nm, connection works again about 1 min.

**Comment by alexeymuranov** [comment]
Related (or duplicate): #72416.

**Comment by alexeymuranov** [comment]
Must be fixed by #72951.
